workshop/.gitignore

```diff
@@ -0,0 +1 @@
+/workshop
```

Comment:
let's make it `workshop/<name of this specific workshop>` please

Reply:
This workshop has been already selected for being repeated in other events across Python Spain Network .

If you are ok with this , we can follow the same mental model as we do for other datasets for  **consistency**  
 and future material **leveraging** , and do `workshop/satellitedata` . 

Let me know your thoughts :)

Reply:
yes, absolutely, `workshop/satellitedata` makes sense to